content/hi/docs/tutorials/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: ट्यूटोरियल
+main_menu: true
+no_list: true
+weight: 60
+content_type: concept
+---
+
+<!-- overview -->
+
+प्रलेखन के इस खंड में ट्यूटोरियल हैं।
+ट्यूटोरियल दिखाता है कि किसी एकल कार्य से बड़े [लक्ष्य](/docs/tasks/) को कैसे पूरा किया जाए।
+आमतौर पर एक ट्यूटोरियल में कई खंड होते हैं,
+जिनमें से प्रत्येक में चरणों के क्रम होते हैं।
+प्रत्येक ट्यूटोरियल से परिचित होने से पहले, हम आपको 
+[मानकीकृत शब्दावली](/docs/reference/glossary/) पृष्ट को बुकमार्क करने की सलाह देते हैं। 
+<!-- body -->
+
+## मूलभूत
+
+* [मूलभूत कुबरनेट्स](/hi/docs/tutorials/kubernetes-basics/) एक गहन इंटरैक्टिव ट्यूटोरियल है जो आपको कुबेरनेट्स प्रणाली को समझने और कुबेरनेट्स की कुछ बुनियादी सुविधाओं को आज़माने में मदद करता है।
```

Comment:
> [मूलभूत कुबरनेट्स](/hi/docs/tutorials/kubernetes-basics/) एक गहन इंटरैक्टिव ट्यूटोरियल है जो आपको कुबेरनेट्स प्रणाली को समझने और कुबेरनेट्स की कुछ बुनियादी सुविधाओं को आज़माने में मदद करता है।

Would संवादात्मक be better here? I know the literal translation is more towards the interactive nature of conversations rather than that of a tutorial.